src/app/static/src/app/components/files/ManageFile.vue

```diff
@@ -1,7 +1,9 @@
 <template>
     <div class="form-group">
         <label class="font-weight-bold mb-0" v-translate="label"></label>
-        <span id="required" v-if="required" :class="existingFileName? '': 'text-danger'">(required)</span>
+        <span id="required" v-if="required"
+              class="small"
+              :class="existingFileName? '': 'text-danger'">({{requiredText}})</span>
```

Comment:
I think it would be simpler here to use the `v-translate` directive than to have to map currentLanguage and add the requiredText property - even though that would mean adding a nested span here so that you could still have the brackets in the outer one.

Reply:
I don't think a nested span is neat and readable.

Reply:
That's what `v-translate` is for though, we have nested spans in lots of other places because that's how it works. It adds less boilerplate than the additional map and property.